tests/robot-tests/tests/admin_and_public/bau/footnotes.robot

```diff
@@ -29,6 +29,7 @@ Create new publication and release via API
 
 Upload subject
     [Tags]  HappyPath
+    user waits for page to finish loading
```

Comment:
It's best not to resort to this keyword if you don't need to and instead wait on something that should appear on the page.

The `user waits for page to finish loading` keyword got created because of a particular scenario where there was no element on the page I could wait on - I think it was something like a radio button that I had waited on already, but it still hadn't loaded the react javascript around it. So I was forced to add a sleep.

All that said I notice this keyword has got used all over the tests now - either way better to use something else if possible.

Reply:
Without this keyword the pipeline seemed to consistently fail on: `Element 'xpath://*[contains(@class,"govuk-accordion__section-button") and text()="UI test subject"]' did not appear in 30 seconds.`